### PR TITLE
Create tstrait.md

### PR DESCRIPTION
**Pull request by daikitag**
Create page for tstrait

### CODE REVIEW

File: _software/tstrait.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  )
+  sim_result = tstrait.sim_phenotype(
+      ts=ts, num_causal=100, model=model, h2=0.3, random_seed=1
+  )
+redirect_from:
+  - /tstrait/
+---
+
+tstrait is a Python package to simulate quantitative traits from a given
+tree sequence data.
```
Comment (jeromekelleher) [inline _software/tstrait.md:25]:
```suggestion
tree sequence data.
```
```suggestion
tree sequence.
```

**Review by jeromekelleher — COMMENTED** [review]
LGTM one suggestion

File: _software/tstrait.md (hunk 1 above)
Comment (daikitag) [reply _software/tstrait.md]: Thank you very much for your suggestion. I really appreciate it.

**Review by benjeffery — APPROVED** [review]
LGTM!

**Pull request merged**